### DISCUSSION TITLE
Move the backup function to the settings page

### DISCUSSION THREAD

**Discussion by quazar-omega** [ncosgray/cuppa_mobile#154] [Ideas]
(Related to #128)
I was looking for the feature to export he data and initially couldn't find it, I clicked on the question mark icon just out of curiosity not expecting to find it there, I think it'd make more sense to keep it with the rest of the settings for better discoverability

**Comment by ncosgray** [comment]
You're so right! I'm not entirely sure how the export/import buttons ended up on the About page instead of on the Prefs page. I will make this change.

Thank you so much for your interest in Cuppa. :)

**Reply by quazar-omega** [reply]
Thank _you_ for the handy app, I've used it so much now to get the timing right on my teas!  
And happy to hear you agree on the suggestion
